### PR TITLE
Update Remove-RDDatabaseConnectionString.md

### PR DESCRIPTION
**Pull request by msbemba**
added example to the document

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1597

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/remotedesktop/Remove-RDDatabaseConnectionString.md [hunk 1]
```diff
@@ -51,11 +51,13 @@ Currently, you cannot use this cmdlet to remove the primary database connection
 
 ## EXAMPLES
```
Comment (JohanFreelancer9) [inline docset/windows/remotedesktop/Remove-RDDatabaseConnectionString.md]:
```suggestion
## EXAMPLE
```

File: docset/windows/remotedesktop/Remove-RDDatabaseConnectionString.md [hunk 2]
```diff
@@ -51,11 +51,13 @@ Currently, you cannot use this cmdlet to remove the primary database connection
...
 
-### 1:
+### 1: Remove a remote desktop database connection string
```
Comment (JohanFreelancer9) [inline docset/windows/remotedesktop/Remove-RDDatabaseConnectionString.md]:
```suggestion
### Remove a remote desktop database connection string
```

**Comment by get-itips** [comment]
@msbemba Please review the build log as it is not currently building without error, thank you

**Comment by msbemba** [comment]
> @msbemba Please review the build log as it is not currently building without error, thank you

Thank you for pointing out. I see it was due to the examples Header which is now corrected.

**Pull request merged**